### DISCUSSION TITLE
Including request example data for `Path()` parameters when they are given through dependency injection

### DISCUSSION THREAD

**Discussion by nleroy917** [fastapi/fastapi#8538] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# main.py
from fastapi import FastAPI, JSONResponse
from .routers import namespace

app = FastAPI()

app.include_router(
    namespace.router
)

# routers/namespace.py
from fastapi import APIRouter, Depends

from ..dependencies import *
from ..main import _PEP_STORES
from ..route_examples import example_namespace

router = APIRouter(
    prefix="/pep/{namespace}",
    dependencies=[Depends(verify_namespace)],
)

@router.get("/", summary="Fetch details about a particular namespace.")
async def get_namespace(namespace: str = example_namespace):
    """Fetch namespace. Returns a JSON representation of the namespace and the projects inside it."""
    return JSONResponse(content=_PEP_STORES[namespace])

# dependencies.py
from .main import _PEP_STORES

def verify_namespace(namespace: str = example_namespace) -> None:
    if namespace not in _PEP_STORES:
        raise HTTPException(status_code=404, detail=f"namespace '{namespace}' not found.")

# route_examples.py
from fastapi import Path

example_namespace = Path(
    ...,
    description="A namespace that holds projects.",
    example="demo",
)
```

### Description

I guess I am having an issue declaring _request data examples_ for specific endpoint/route parameters when said endpoint/route parameters _are defined through dependency injection_. In the above example, I need to verify that the namespace actually exists prior to returning data about it. To do that, I followed [this example](https://fastapi.tiangolo.com/tutorial/dependencies/global-dependencies/) in the FastAPI docs. It works great.

However, in `/docs`, I'd love to give examples for namespaces. To try and do that, I followed [this example](https://fastapi.tiangolo.com/tutorial/schema-extra-example/) in the FastAPI docs.

Visiting `/docs`, however, provides the user with no example request data:
![image](https://user-images.githubusercontent.com/41063083/148662055-af85bf90-61aa-4384-98a9-0a787b6511f3.png)

If I comment out the global dependencies like so:

```python
router = APIRouter(
    prefix="/pep/{namespace}",
    # dependencies=[Depends(verify_namespace)],
)
```

The example is shown. I can also verify that declaring requirements in the `Path()` instance like `min_length` or `max_length`, these requirements are honored by FastAPI (even though I can't see the requirements in `/docs`).

So, I guess my question is the title: **How do you declare request data for route parameters when the route parameter is defined through dependency injection**

### Operating System

Other

### Operating System Details

macOS/Linux

### FastAPI Version

0.71.0

### Python Version

3.8

### Additional Context

Running inside Docker: `FROM tiangolo/uvicorn-gunicorn:python3.8`

**Comment by YuriiMotov** [comment]
Seems that it works fine in current version (0.115.13)
